src/test/resources/features/refapp-2.x/03-trackPatient/editDemographicsAndContactInfo.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Edit Demographics and contact info from Patient Dashboard
+
+  Background:
+    Given a user clicks on Edit Registration Information link from Patient dashboard
+    Then the system loads Registration Summary Page
+
+  @selenium
+  @patientDemographics
+  Scenario: Edit Patient Demographics and Contact Info
+    # User story: Edit Patient Demographics
+    When a user clicks on Edit link from Registration Summary Page
+    And the system loads edit demographics section
+    And a user edits demographics
+    And a user clicks on the confirm button
+    Then the system saves the updated patient demographics
+    
+    # User story: Edit Patient Contact Information
+    When a user clicks on Edit link under contact info section from Registration Summary Page
+    And the system loads the edit contact information section
```

Comment:
isn't this gherkin step supposed to be 
```suggestion
    Then the system loads the edit contact information section
```
?

Reply:
@kdaud I am getting a bit confused. At some point I wrote my steps that way and you advised me to change and follow the When And And And And Then way of writing  and you gave me this example [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/153#discussion_r731482251) . However, now you are advising me write it differently. You actually asked me to look into this[ link ](https://support.smartbear.com/cucumberstudio/docs/tests/best-practices.html). What could be the best procedure to follow.

Reply:
> I wrote my steps that way

@Irenyak1 then you're right! However, have you revisited the [recommendation](https://github.com/openmrs/openmrs-contrib-qaframework/pull/153#discussion_r731482251)?

Reply:
@kdaud sorry I mistakenly deleted my comment here but I believe you have it in your email. 
I await your response. But I've also realized that you have edited the initial steps that you had sent to my PR as review. 
I think I get the reason why.